whipper/command/main.py

```diff
@@ -20,7 +20,13 @@
 
 def main():
     server = config.Config().get_musicbrainz_server()
-    musicbrainzngs.set_hostname(server)
+    https_enabled = True if server['scheme'] == 'https' else False
```

Comment:
This can just be: `https_enabled = 'server['scheme'] == 'https'`

Reply:
Done, thanks!